pkgs/top-level/aliases.nix

```diff
@@ -601,6 +601,7 @@ mapAliases ({
   hardlink = throw "hardlink was merged into util-linux since 2019-06-14."; # Added 2022-08-12
   inherit (harePackages) hare harec; # Added 2022-08-10
   hawkthorne = throw "hawkthorne has been removed because it depended on a broken version of love"; # Added 2022-01-15
+  hdr-plus = throw "hdr-plus has been removed: unmaintained, original reason for inclusion is no more"; # Added 2022-11-08
```

Comment:
```suggestion
  hdr-plus = throw "hdr-plus has been removed because it unmaintained, often breaks and no longer consumed as a dependency"; # Added 2022-11-08
```

Reply:
Done.